projet/projects/models.py

```diff
@@ -29,6 +30,52 @@ def save(self, *args, **kwargs):
     def count_contributors(self):
         return self.contributors.count()
 
+    def total_expenditure(self):
+        try:
+            expenses = Expense.objects.filter(project__id=self.id)
+            total = 0
+            for amount in expenses:
+                total += amount.amount.amount
+            return total
+        except Expense.DoesNotExist:
+            return ""
+
+    def over_budget_by(self):
+        try:
+            budget = Budget.objects.get(project__id=self.id)
```

Comment:
Calculates how much the project is over budget by, by getting all of the projects expenses and adding it's amount.